python/mim_control/robot_centroidal_controller.py

```diff
@@ -1,10 +1,3 @@
-##################################################################################################################
```

Comment:
Any reason for deleting this? You can use a different format if you want it to look like others.

Reply:
Not sure what happened, I added it back.